### ISSUE TITLE
Lags without reason

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/Unvanquished#1606]
The problem occurred now multiple times also with different players, myself included.
Players will just start lagging without any reason. The ping is low or constant, fps is normal and graphic settings have no influence.
Its also visible for other players as they see the lag while spectating or the game starts to lag if they get close to the affected one.
That's another oddity it seems to always only affect one player. 
Restarting the game or changing settings has also no effect.

**Comment by bmorel** [comment]
I confirm having been affected by it multiple times, on various servers. Last time was on gireen's. I am unsure, but I think I have been affected on my servers too.

I do not know if I have seen it on illlwieckz or ITA ones.

I rarely play on official server since I'm laggy there anyway (probably physical distance).

**Comment by slipher** [comment]
How do you know there's not a network issue? You can have packet loss without changes in latency.

What do you mean by "lagging"?

**Comment by bmorel** [comment]
The symptoms I had (more than once) are: when close to a player, the entity teleports between places. The thing we usually call a lag.
Prediction seems to not happen in those cases, because said entity also seems to freeze. This is pure feeling, though, it's hard to describe when not seeing it, and YMMV.

Yes, it might be a network issue, but the strange thing is, it happens when a specific entity (a player) is on screen, when said entity is dead or far enough, it does not happens.
It might be caused by packet drops, though, but I have no idea how to have that kind of statistics.

**Comment by Gireen** [comment]
Yes its classical lag. Teleporting, stuttering movements, invisible obstacles for a few ms.

```
^5Debug: Noob123^*: Delta request from out of date packet.
```
Is this packet loss? In this case this would be an explanation in one case.

In the event that freem describes there was this message only from another player.
```
^5Debug: manoliz^*: Delta request from out of date packet.
^5Debug: manoliz^*: Delta request from out of date packet.
^5Debug: manoliz^*: Delta request from out of date packet.
^5Debug: manoliz^*: Delta request from out of date packet.
^5Debug: clientCommand: freem^* : 338 : say "I'm laggy :/"
```
But it did seem only to happen around freem, with such a strong effect that one could tell who he was in a group 😄

**Comment by bmorel** [comment]
Note that I didn't had the problem when nobody was around, though.

**Comment by bmorel** [comment]
I think this happens each time some player have high ping: it makes everyone lag, as if server were trying to wait from the laggy person before sending stuff
